goodmap/goodmap.py

```
@@ -87,6 +89,43 @@ def create_app_from_config(config: GoodmapConfig) -> platzky.Engine:
+                    response.headers["Access-Control-Allow-Origin"] = "*"
+                    return response
+
+                app.register_blueprint(bp)
+                plugin_manifest.append(
+                    {
+                        "scope": ep.name,
+                        "url": f"/plugins/{ep.name}/static/remoteEntry.js",
+                        "module": "./Button",
+                    }
+                )
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)